x/liquidity/spec/04_messages.md

```diff
@@ -1,87 +1,109 @@
-<!--
-order: 4
--->
+<!-- order: 4 -->
 
 # Messages
 
+Messages (Msg) trigger state transitions. Msgs are wrapped in transactions (Txs) that clients submit to the network. The Cosmos SDK wraps and unwraps Liquidity Module messages from transactions.
+
+All Liquidity Module messages require a corresponding handler that performs validation logic. See [State Transitions](./03_state_transitions).
```

Comment:
I don't know if all messages require a handler, but maybe this is the state transitions?

Reply:
no response, so closing this suggestion and removing this content